src/main/java/apache/ranger/client/utils/Base64.java

```diff
@@ -0,0 +1,42 @@
+package apache.ranger.client.utils;
+
+public class Base64 {
```

Comment:
Can't we use Java's [Base64](https://docs.oracle.com/javase/8/docs/api/java/util/Base64.html) encoder?

Reply:
they are diffrent encode methods